style.css

```diff
@@ -343,33 +396,261 @@ input:focus {
     border-radius: 6px;
     height: auto;
 }
+
 .cont2_a li:hover .submenu-cont2a2 ul {
     display: block;
     padding: 0;
     margin: 0;
     text-align: start;
 }
+
 .cont2_a li:hover .submenu-cont2a2 li {
     list-style: none;
     Padding: 15px 20px;
 }
+
 .cont2_a li:hover .submenu-cont2a2 ul li a:hover {
     border-bottom: none;
     border: 1px solid lightgray;
     border-radius: 6px;
     padding: 8px 6px;
     background-color: lightgray;
 }
-.submenu-cont2a2 ul li a:hover .find:hover {
+
+.submenu-cont2a2 li:hover {
+    cursor: pointer;
+}
+
+.submenu-cont2a3 {
+    display: none;
+}
+
+.cont2_a li:hover .submenu-cont2a3 {
+    display: block;
+    position: absolute;
+    margin-top: 23px;
+    width: 190px;
+    border: 2px solid white;
+    background-color: white;
+    border-radius: 6px;
+}
+
+.cont2_a li:hover .submenu-cont2a3 ul {
+    text-align: left;
+    display: block;
+}
+
+.cont2_a li:hover .submenu-cont2a3 ul li {
+    list-style: none;
+    margin: 25px 20px;
+}
+.cont2_a li:hover .submenu-cont2a3 li a:hover {
     text-decoration: none;
     border-bottom: none;
-    border: none;
-    border-radius: 0;
-    padding: 0;
+    border: 1px solid lightgray;
+    border-radius: 6px;
+    padding: 8px 6px;
+    background-color: lightgray;
+}
+.submenu-cont2a4 {
+    display: none;
+}
+.cont2_a li:hover .submenu-cont2a4 {
+    display: flex;
+    flex: 40% 40% 20%;
+    gap: 10px;
+    position: absolute;
+    margin-top: 23px;
+    width: 450px;
+    border: 2px solid white;
     background-color: white;
+    border-radius: 6px;
 }
-.submenu-cont2a2 li:hover {
-    cursor: pointer;
+.cont2_a li:hover .submenu-cont2a4 ul {
+    display: block;
+    text-align: left;
+}
+.cont2_a li:hover .submenu-cont2a4 ul li {
+    list-style: none;
+    margin: 25px 15px;
+}
+.cont2_a li:hover .submenu-cont2a4 ul li a:hover {
```

Comment:
Could you use a definitive class name.

Reply:
Thank you @lillian-busingye

I have changed the class names.